python/adelphi/integration-tests/adelphi-docker-tests.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+exitCode=0
+
+function start() {
+	local version=$1
+	echo "Starting C* $version"
+	docker run --name adelphi --network bridge --rm -d -p 9042:9042 cassandra:$version >> logs/$version.log
+	wait
+	echo "Cassandra ready"
+}
+
+function stop() {
+	local version=$1
+	echo "Stopping C* $version"
+	docker stop adelphi >> logs/debug-$version.log
+}
+
+function wait() {
+	printf "Waiting for server"
+	until docker exec -it adelphi cqlsh -e "select * from system.local" >> logs/$version.log 
+	do
+		printf "."
+		sleep 1
+	done
+	printf "\n"
+}
+
+function create_schema() {
+	local version=$1
+	docker exec -it adelphi cqlsh -e "`cat base-schema.cql`" >> logs/$version.log
+}
+
+function anonymize() {
+	local version=$1
+	echo "Anonymizing..."
+	adelphi export-cql > output/$version.cql 2>> logs/$version.log
+}
+
+function compare() {
+	local version=$1
+	echo "Comparing..."
+	diff schemas/$version.cql output/$version.cql
+	diffExitCode=$?
```

Comment:
keeps track of the exit code, such that even if the last comparison succeeded, but some previous comparison failed, we want the parent script to fail